core/src/message/context.rs

```diff
@@ -34,7 +34,9 @@ pub const OUTGOING_LIMIT: u32 = 1024;
 /// Context settings.
 #[derive(Copy, Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, Decode, Encode, TypeInfo)]
 pub struct ContextSettings {
+    /// Amount of gas needed for send a message.
```

Comment:
That's fee that we gonna take for message sending

Reply:
Fixed it.